components/node-labeler/cmd/run.go

```diff
@@ -226,9 +226,9 @@ func (r *PodReconciler) Reconcile(ctx context.Context, req reconcile.Request) (r
 	}
 
 	if component == registryFacade {
-		err = waitForRegistryFacade(ipAddress, port, 30*time.Second)
+		err = checkegistryFacade(ipAddress, port)
 		if err != nil {
-			return reconcile.Result{}, fmt.Errorf("waiting for registry-facade: %v", err)
+			return reconcile.Result{RequeueAfter: time.Second * 10}, err
```

Comment:
`RequeueAfter` gets ignored if `err` is non nil, and will use exponential backoff in that case, not sure if that's what we want?

Reply:
changed